File: app/src/main/java/com/microlabs/trallet/view/CurrencyActivityView.java
package com.microlabs.trallet.view;

import com.microlabs.trallet.model.Currency;

import java.util.List;

/**
 * Created by Hansen on 5/11/2017.
 *
 * Currency Activity View
 */

public interface CurrencyActivityView {

    void showCurrencyList(List<Currency> currencyList);
}

File: app/src/main/java/com/microlabs/trallet/view/CurrencyActivityView.kt
package com.microlabs.trallet.view

import com.microlabs.trallet.model.Currency

/**
 * Created by Hansen on 5/11/2017.
 *
 * RCurrency Activity View
 */

interface CurrencyActivityView {

    fun showCurrencyList(RCurrencyList: List<Currency>)
}
